cypress/integration/articleFlows/commentOnArticle.spec.js

```diff
@@ -202,7 +202,7 @@ describe('Comment on articles', () => {
       cy.findByText('@search_user_1').should('not.be.visible');
     });
 
-    it('should reply to a comment with user mention autocomplete', () => {
+    xit('should reply to a comment with user mention autocomplete', () => {
```

Comment:
Should we leave a comment or TODO here explaining why it's skipped for now?

Reply:
I'm going to revisit this later today/tomorrow.